api/secprofnodestatus/v1alpha1/secprofnodestatus_types.go

```diff
@@ -63,9 +65,10 @@ func LowerOfTwoStates(currentLowest, candidate ProfileState) ProfileState {
 	orderedStates := make(map[ProfileState]int)
 	orderedStates[ProfileStateError] = 0       // error must always have the lowest index
 	orderedStates[ProfileStateTerminating] = 1 // If one is set as terminating; all the statuses will end here too
-	orderedStates[ProfileStatePending] = 2
-	orderedStates[ProfileStateInProgress] = 3
-	orderedStates[ProfileStateInstalled] = 4
```

Comment:
Since this is a public method, does changing the mapping between the state and the integer matter?